.github/workflows/update_version.yml

```diff
@@ -0,0 +1,42 @@
+name: Update the npm package version
+
+on:
+  workflow_dispatch:
+    inputs:
+        version:
+          required: true
+          type: choice
+          description: Version type
+          default: patch
+          options: 
+          - major
+          - minor
+          - patch
+          - premajor
+          - preminor
+          - prepatch
+          - prerelease
+jobs:
+  update_version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 19
+          cache: 'npm'
+      - id: bump_version
+        run: |
+          echo "old_version=$(jq -r .version package.json)" >> $GITHUB_OUTPUT
+          npm version ${{ github.event.inputs.version }} --git-tag-version false
```

Comment:
Are you sure that's supported? I was looking in the npm docs and don't see that mentioned
https://docs.npmjs.com/cli/v8/commands/npm-version#git-tag-version

It might be supported, but deprecated

Reply:
> If run in a git repo, it will also create a version commit and tag. This behavior is controlled by git-tag-version (see below), and can be disabled on the command line by running npm --no-git-tag-version version. It will fail if the working directory is not clean, unless the -f or --force flag is set.

from [v9 docs](https://docs.npmjs.com/cli/v9/commands/npm-version)

Reply:
Ahhh, yep I was looking at the wrong docs, thanks!